src/app/plan/duck/selectors.ts

```diff
@@ -64,6 +66,13 @@ const getPlansWithStatus = createSelector(
               return m.status.conditions.some(c => c.type === 'Running');
             }
           }).length;
+
+          if (plan.Migrations[0].status.conditions.length) {
+            latestMigrationStep = plan.Migrations[0].status.conditions.filter(c => {
+              return c.type === 'Running';
+            })[0];
+            console.log(latestMigrationStep = plan.Migrations[0].status.conditions.filter(c => { return c.type === 'Running' })[0])
```

Comment:
Thinking there's going to be a better way of expressing this cleanly without referencing the 0th index that I don't know exists.

Reply:
Using the us find method for this might work